packages/next-app/src/components/Header.tsx

```diff
@@ -10,17 +11,19 @@ interface IHeader {
   linkUrl: UrlObject | __next_route_internal_types__.RouteImpl<string>;
   imageSrc?: string;
   title: string;
+  children?: ReactNode;
```

Comment:
Prefer to extend `React.PropsWithChildren`

Reply:
but if the header does not have children?

Reply:
No problem, this is how the type is defined:
![Uploading Screenshot 2023-04-05 at 08.51.47.png…]()